### ISSUE TITLE
Test failures due to signer information mismatch

### ISSUE THREAD

**Issue by yudistrange** [JOSM/Mapillary#68]
The tests for the Mapillary plugin are failing with following exception for multiple classes. 
`class "org.openstreetmap.josm.tools.MemoryManagerTest"'s signer information does not match signer information of other classes in the same package`

How to reproduce:
`./gradlew test`

**Comment by floscher** [comment]
I fixed that issue in 1f1c8178b658cb57a8d74967a2561c9f365a0eaa, but then hit another one:
On my machine everything is fine, but when building on Travis CI, it complains:
```
java.lang.NoClassDefFoundError: Could not initialize class org.openstreetmap.josm.plugins.mapillary.MapillaryLayer
```
(https://travis-ci.org/JOSM/Mapillary/jobs/292282737#L802)

I'm investigating, but currently don't know what's causing this.

**Comment by floscher** [comment]
I believe I found the cause of this issue. The fix will probably land tomorrow. :sleeping:
